CONTRIBUTING.md

```diff
@@ -157,6 +157,8 @@ This tag is for boolean properties associated with the test.
   created procedurally using the project's test generation tool; refer to the
   section titled "Procedurally-generated tests" for more information on this
   process
+- **`CanBlockIsFalse`** - only run the test when the [[CanBlock]] property of the [Agent Record](https://tc39.github.io/ecma262/#sec-agents) executing the test file is `false`
+- **`CanBlockIsTrue`** - only run the test when the [[CanBlock]] property of the [Agent Record](https://tc39.github.io/ecma262/#sec-agents) executing the test file is `true`
```

Comment:
Thanks for this. While this is correct for the current model, I believe it would be easier to move this flag to the API on $262.agent. Perhaps, this improves communicating with the engine running the test. While the flag would add more separation in the test runners.

What do you think? We can have it fixed here or in a follow up, if you're positive with this new idea. Otherwise, the current changes here are the best.

Reply:
I guess it depends on the test runner. For example for SpiderMonkey/Firefox, a separate flag is easy to implement and allows us to reuse existing machinery to differentiate between shell-only ([[CanBlock]] is true) and browser-only ([[CanBlock]] is false) tests. That being said, we don't yet run Atomics tests in the browser, because our `$test262.agent` implementation uses shell-only features. Therefore it's a bit difficult to say which approach works best (flags vs. `$262.agent` API), at least from the perspective of SpiderMonkey/Firefox.

Reply:
The flag is already set, so I'm thinking on trying that with runners anyway. We can reconsider using the agent later if we find any arguments.